.github/workflows/images.yml

```diff
@@ -22,7 +23,7 @@ on:
       - '**.jpeg'
       - '**.png'
       - '**.webp'
-  workflow_dispatch:
```

Comment:
If this was removed on purpose, please also update the comment at the beginning of the file. That also still mentions that the workflow is run on schedule.